src/core/utils/DefaultKeyMap.ts

```diff
@@ -0,0 +1,53 @@
+import { VNodeType } from '../stores/VNode';
+
+const map = new Map();
+const shortcuts = {
+    'ctrl+alt+&': {
+        name: 'formatParagraph',
+        arguments: [VNodeType.HEADING1],
+    },
+    'ctrl+alt+é': {
+        name: 'formatParagraph',
+        arguments: [VNodeType.HEADING2],
+    },
+    'ctrl+alt+"': {
+        name: 'formatParagraph',
+        arguments: [VNodeType.HEADING3],
+    },
+    "ctrl+alt+'": {
+        name: 'formatParagraph',
+        arguments: [VNodeType.HEADING4],
+    },
+    'ctrl+alt+(': {
+        name: 'formatParagraph',
+        arguments: [VNodeType.HEADING5],
+    },
+    'ctrl+alt+§': {
+        name: 'formatParagraph',
+        arguments: [VNodeType.HEADING6],
+    },
+    'ctrl+alt+à': {
+        name: 'formatParagraph',
+        arguments: [VNodeType.PARAGRAPH],
+    },
+    'ctrl+b': {
+        name: 'applyFormat',
+        arguments: {
+            format: 'bold',
+        },
+    },
+    'ctrl+i': {
+        name: 'applyFormat',
+        arguments: {
+            format: 'italic',
+        },
+    },
+    'ctrl+u': {
```

Comment:
I just checked in Google Drive on MacOS and in Sublime Text on Linux and both use the following typography: `Modifier+K`

That is the first letter of the modifier is capitalized and the actual key is capitalized as well, even though you're not supposed to type the capitalzed letter. So `Ctrl+K` triggers on `Ctrl+k`, otherwise it would require `Ctrl+Shift+k` to truly type `Ctrl+K`. In fact, I just realized the letters are capitalzed on the keyboards, so pressing the `K` key actually enters `k`, you need `Shift+K` to truly insert `K`, funny.

I still have a personal preference for all capitalized shortcuts like `CTRL+SHIFT+K` though, because somehow seeing it that way makes it more obvious in my brain that each of them are a key of their own, as they are like a "constant" of the keyboard if you like. The cognitive links in my brain activates faster when I see them all capitalized, but maybe it's just me. Both `Ctrl+Shift+K` and `CTRL+SHIFT+K` are acceptable to me, but not `ctrl+shift+k`.

If you do not like "all caps" then I suggest to use `Ctrl+Shift+K` in plugin definitions, and to `toUpperCase` it for the system rather than 'toLowerCase' like it is now. The system itself won't really care anyway... :p

Reply:
Yeah I tried a few different things and the lowercase felt "lightest" but that may have been cause I was starting to come down with the flu and felt uppercase letters were aggressing me 😅 Using whatever is standard definitely makes sense.